.gitignore

```diff
@@ -1,6 +1,8 @@
 # Project
 build/
 WienerTakesAll
+lib
+lib.zip
 
```

Comment:
Might as well add the ```physx/``` directory to the gitignore so that it doesn't get accidentally committed to the repo, too!